source/PlayState.hx

```diff
@@ -1313,7 +1315,9 @@ class PlayState extends MusicBeatState
 	
 		#if desktop
 		// Updating Discord Rich Presence.
-		DiscordClient.changePresence(detailsText, SONG.song + " (" + storyDifficultyText + ")", iconP2.getCharacter());
+		var setIcon:HealthIcon = iconP2;
+		if (opponentPlay) {setIcon = iconP1;} else {setIcon = iconP2;}
```

Comment:
You could just do 
`var setIcon:HealthIcon = (opponentPlay ? iconP1 : iconP2);`

Reply:
Oh... I didn't know that so thx!